test/freenet/support/MultiValueTableTest.java

```diff
@@ -234,8 +233,8 @@ public void testGetSync() {
 		Object[][] sampleObjects = 
 			createSampleKeyMultiVal(sampleKeyNumber,sampleMaxValueNumber,sampleIsRandom);
 		methodMVTable = fillMultiValueTable(sampleObjects);
-		for(int i=0;i<sampleObjects.length;i++)
-			assertEquals(methodMVTable.getSync(sampleObjects[i][0]),((List<Object>)sampleObjects[i][1]));
+		for (Object[] sampleObject : sampleObjects)
```

Comment:
Cast `sampleObjects`, change the type of `sampleObject`, remove cast in line below. And braces, please.